ibis/spark/client.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import pyspark
+from pyspark import SparkConf, SparkContext
+from pyspark.sql import SparkSession
+from pyspark.sql.types import (BooleanType, NullType, ArrayType,
+                               StringType, BinaryType, DateType,
+                               TimestampType, ShortType, IntegerType,
+                               LongType, FloatType, DoubleType,
+                               DecimalType, StructType)
+
+import ibis.client as client
+import ibis.expr.datatypes as dt
+import ibis.expr.operations as ops
+
+from ibis.compat import parse_version
+
+
+_ibis_dtypes = {
+    dt.Boolean: BooleanType,
+    dt.Null: NullType,
+    dt.Array: ArrayType,
+    dt.String: StringType,
+    dt.Binary: BinaryType,
+    dt.Category: 'category',  # FIXME
```

Comment:
You can leave this out if you want. `Category` isn't well supported in ibis yet.